applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.cpp

```diff
@@ -1080,45 +1045,31 @@ namespace Kratos
 
 
 		bounded_matrix<double,msElementSize,msElementSize> aux_matrix = ZeroMatrix(msElementSize);
-		aux_matrix = prod(TransformationMatrix, rLeftHandSideMatrix);
+		aux_matrix = prod(transformation_matrix, rLeftHandSideMatrix);
 		rLeftHandSideMatrix = prod(aux_matrix,
-			Matrix(trans(TransformationMatrix)));
+			Matrix(trans(transformation_matrix)));

```

Comment:
you can use the trans directly here, without allocating the matrix.

matrix allocation is need only in the triple product